lib/cisco_node_utils/bgp_af.rb

```diff
@@ -142,7 +170,6 @@ def next_hop_route_map
     end
 
     def next_hop_route_map=(route_map)
-      route_map.strip!
```

Comment:
I suggested removing the `route_map.strip!` and adding `kind: string` to the YAML entry because `config_get` would automatically strip the white space.